src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -1386,6 +1413,21 @@ module Mutations = struct
       ~args:Arg.[arg "input" ~typ:(non_null Types.Input.create_account)]
       ~resolve:create_account_resolver
 
+  let register_hardware_wallet : (Coda_lib.t, unit) field =
+    io_field "registerHardwareWallet"
```

Comment:
maybe should be called `createHardwareWalletAccount` or `createAccountHardwareWallet`
cc @o1pranay for naming thoughts

Reply:
I discussed with pranay/carey and it seems like if everyone's on board, we can go with "createHDAccount", since it seems like that describes the protocol used by the ledger <https://ledger.readthedocs.io/en/latest/background/hd_keys.html> and lets us be a little future-proof in the API